x/subscription/migrations/v4/keys.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "subscription"
```

Comment:
why do we need the keys if they didnt change?

Reply:
but they could change in the future. I want the migration to be fully standalone